mr/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -172,5 +221,18 @@ public String identifier(String tableIdentifier) {
     public Map<String, String> properties() {
       return ImmutableMap.of(InputFormatConfig.CATALOG, "hive");
     }
+
+    @Override
+    public String locationForCreateTableSQL(TableIdentifier identifier) {
+      return "";
```

Comment:
Empty string defaults are very suspicious to me. Why not use null here?

Reply:
I used this to add location string to the Hive test queries.
I wanted to avoid writing every time:
```
(location==null ? "" : location)
```